plugins/ompgsql/ompgsql.c

```diff
@@ -364,8 +360,14 @@ CODE_STD_STRING_REQUESTparseSelectorAct(1)
 		iPgSQLPropErr++;
 	if(*pData->f_dbuid == '\0')
 		iPgSQLPropErr++;
-	if(getSubString(&p, pData->f_dbpwd, _DB_MAXPWDLEN+1, ';'))
+	if(getSubString(&p, pData->f_dbpwd, _DB_MAXPWDLEN+1, ','))
 		iPgSQLPropErr++;
+	if(*pData->f_dbpwd == '\0')
+		iPgSQLPropErr++;
+	if(getSubString(&p, pData->f_dbsrvport, _DB_MAXDBSRVPORTLEN+1, ';'))
```

Comment:
This looks like it still requires the parameter and does not gracefully handle it's absence. So it still breaks backward compatibility.

Reply:
ping?

Reply:
see my previous comment - I think it still applies. Did you also test without the new parameter? Unfortunately we have no testbench test for this.